etc/geoserver/workspaces/MobiData-BW/styles/mdbw_transit_stops_default.sld

```diff
@@ -64,6 +72,10 @@
                 <Fill>
                   <CssParameter name="fill">#7f8c8d</CssParameter>
                 </Fill>
+<Stroke>
+                     <CssParameter name="stroke">#000000</CssParameter>
+		                       <CssParameter name="stroke-width">0.5</CssParameter>
+				                       </Stroke>
               </Mark>

```

Comment:
We did not use `xmllint` until now, but to keep a consistent formatting, I suggest to reformat at least this sld with e.g. 

`xmllint --output mdbw_transit_stops_default.sld --format mdbw_transit_stops_default.sld`.

Note to self: For other slds and xml files, we should lint them in an explicit commit and add linting to the github actions.